Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "sfz"
-version = "0.7.1"
+version = "0.7.2"
```

Comment:
Haven't yet released 0.7.1 so the version needn't to bump
```suggestion
version = "0.7.1"
```

Reply:
Rolled back. Please help review thanks.